uint8 connected
float32 distance
float32 offsetRight
float32 offsetLeft
